README.md

```diff
@@ -291,6 +291,116 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+  <p>
```

Comment:
This still isn't [rendering](https://github.com/paulmolluzzo/glamorous/blob/d65f17fcc319ab75ca626e67d4eff1b6121363ac/README.md) quite right. I think what you're looking for is this:

```
<details>
<summary>pseudo-class</summary>

```javascript
const MyLink = glamorous.a({
  ':hover': {
    color: 'red'
  }
})

// Use in a render function

<MyLink href="https://github.com">GitHub</MyLink>
```
</details>
```